common/presenters/event-to-tag-component.js

```diff
@@ -0,0 +1,23 @@
+const eventHelpers = require('../helpers/events/event')
+
+const eventToTagComponent = (event, moveId) => {
+  const { id } = event
+
```

Comment:
Any reason for the line between all these declarations? It's not something we've done in many other places.

Reply:
eslint rules pleeze 🙏